### ISSUE TITLE
Default value ignored

### ISSUE THREAD

**Issue by matt-tingen** [adelsz/inquirer-fuzzy-path#3]
It's not clear to me how one would invoke the "default" behavior from the prompt since pressing enter immediately is going to select `rootPath`.  Looking at the code, it looks like the default is just passed on to `inquirer`.  is there some method for using the default value from the prompt that I'm missing?

The question i'm using is:
```js
{
        type: "path",
        name: "path",
        message: "Directory",
        itemType: "directory",
        rootPath: "src/app",
        default: "src/app/shared/components"
}
```

I've also tried `default: 'shared/component'` (relative to `rootPath`).

I'm not exactly sure what expected behavior would be. Perhaps it could list the default value as the first option with the root as the second, and so on e.g.
```
> src/app/shared/components
  src/app
  src/app/foo
  src/app/bar
  src/app/baz
```

**Comment by cgousley** [comment]
bump. Same issue here.

**Comment by GiladShoham** [comment]
same here

**Comment by adelsz** [comment]
Thanks for reporting this!
Fixed this in version 2.0.3.
